README.md

```diff
@@ -1,4 +1,4 @@
 ## This is my Hello Microverse' project
 

```

Comment:
- [ ] This project required having a header with the text content of "Hello Microvers" which is nicely done 👏 However, this project is missing the styling for that header in the CSS file, kindly define a class with styles for your header and make it in your favorite color ✨.